infrastructure-formulas.spec

```diff
@@ -121,6 +123,14 @@ Requires:       %{name}-common
 %description -n grains-formula
 Salt state for managing grains.
 
+%package -n hosts-formula
+Summary:        Salt states for managing %{_sysconfdir}/hosts
+License:        GPL-3.0-or-later
+Requires:       %{name}-common
+
+%description -n hosts-formula
+Salt states for managing the %{_sysconfdir}/hosts file.
+
```

Comment:
hosts-formula looks unrelated to this PR - did you intentionally add it (and its %files)?

Reply:
I just run packaging/genspec.py.

Reply:
I usually do that once for every release, but figured adding this Python functionality warrants an extraordinary run.

Reply:
I will drop it and have it part of the release commit as usual.